redis/conn.go

```diff
@@ -213,6 +232,15 @@ func DialURL(rawurl string, options ...DialOption) (Conn, error) {
 		return nil, fmt.Errorf("invalid database: %s", u.Path[1:])
 	}
 
+	if u.Scheme == "rediss" {
+		// insert a default DlialTLS at position 0, so we don't override any
+		// user provided *tls.Config elsewhere in options
+		t := DialTLS(&tls.Config{InsecureSkipVerify: true})
```

Comment:
No. This because we use self signed certs and even if we didn't we would have to restart stunnel (breaking connections) to rotate the cert. Most customers find this objectionable. We've done the same thing with postgres for years.

With that said, if you are set on not having InsecureSkipVerify set then I can make that change. I may follow up later with a pr to add it as a url encoded option as well.